MafiaAPI/Controllers/MessageController.cs

```
@@ -63,17 +79,40 @@ public JsonResult GetAllMessagesFrom(long id)
         [HttpPost]
         public JsonResult SendMessage(Message message)
         {
+            message.Subject = _securityService.Encrypt(message.Subject);
             message.Content = _securityService.Encrypt(message.Content);
+            message.ReceiveDate = DateTime.Now;
             _messageRepository.Create(message);
             return new JsonResult("Added successfully");
         }
 
+        [Route("/messageContent/{id}")]
+        [HttpGet("{id}")]
```

This should throw an exception because there are two endpoints with the same name now, or even more.  Try use `localhost:xxxx/message/1` in your web browser when backend server is running and see what happen. Coming into conclusion, this {id} annotation is not necessary here.